catkin_tools/verbs/catkin_build/build.py

```diff
@@ -732,12 +733,24 @@ def set_error_state(error_state):
                     # Print them in order of started number
                     for job_msg_args in sorted(executing_jobs, key=lambda args: args['number']):
                         msg += clr("[{name} - {run_time}] ").format(**job_msg_args)
-                    msg_rhs = clr("[{0}/{1} Active | {2}/{3} Completed]").format(
-                        len(executing_jobs),
-                        len(executors),
-                        len(packages) if no_deps else len(completed_packages),
-                        total_packages
-                    )
+
+                    if MakeJobServer.get_instance().supported:
+                        msg_rhs = clr("[{0}/{1} Jobs][{2}/{3} Packages][{4}/{5} Completed]").format(
```

Comment:
This breaks the colorization of the part of the status message. You'll need to provide a matching entry in this file:

https://github.com/catkin/catkin_tools/blob/0afc6f9c21c933348e48a3b3f2da1da63d2a4865/catkin_tools/verbs/catkin_build/color.py#L57-L58